android/clientmanager/src/main/java/io/mosip/registration/clientmanager/dao/RegistrationDao.java

```diff
@@ -40,4 +40,10 @@ public interface RegistrationDao {
     @Insert
     void insert(Registration registration);
 
+    @Query("SELECT * FROM registration where client_status = :status order by cr_dtimes desc")
+    List<Registration> findAllRegistrationByStatus(String status);
+
+    @Query("SELECT * FROM registration where client_status in (:syncedStatus, :approvedStatus) order by cr_dtimes desc")
```

Comment:
since we are displaying count of registration, can we just get that instead of the entire object,
also can we rename the method as findRegistrationCountBySyncedStatusAndApprovedStatus

Reply:
I have made changes